src/llm/http_llm_calculator.cc

```diff
@@ -502,15 +506,15 @@ class HttpLLMCalculator : public CalculatorBase {
             // Unary scenario
             if (!this->request->isStream()) {
                 OVMS_PROFILE_SCOPE("Unary generation cycle");
-                std::vector<GenerationOutput> generationOutput = this->generationHandle->read_all();
+                std::vector<ov::genai::GenerationOutput> generationOutput = this->generationHandle->read_all();
                 RET_CHECK(generationOutput.size() >= 1);
-                std::sort(generationOutput.begin(), generationOutput.end(), [=](GenerationOutput& r1, GenerationOutput& r2) {
+                std::sort(generationOutput.begin(), generationOutput.end(), [=](ov::genai::GenerationOutput& r1, ov::genai::GenerationOutput& r2) {
                     return r1.score > r2.score;
                 });
                 // legacy
                 if (generationOutput.size() == 1) {
                     std::vector<int64_t> tokens = generationOutput[0].generated_token_ids;
-                    std::shared_ptr<Tokenizer> tokenizer = nodeResources->cbPipe->get_tokenizer();
+                    std::shared_ptr<ov::genai::Tokenizer> tokenizer = std::make_shared<ov::genai::Tokenizer>(nodeResources->cbPipe->get_tokenizer());
```

Comment:
This creates copy of tokenizer in each iteration. It was not copied before this change.
To avoid the copy, my suggestion would be to save the tokenizer for later use in first pass.